app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragment.kt

```diff
@@ -40,7 +40,7 @@ class NavigationDrawerFragment :
     navigationDrawerFragmentPresenter.openProfileProgress(profileId)
   }
 
-  override fun markHomeMenuCloseDrawer() {
+  override fun restoreLastCheckedMenuItem() {
```

Comment:
instead of restore maybe we can go with 
`markPreviousMenuItemCloseDrawer`

Reply:
Maybe we want to append `CloseDrawer` in the name.